src/Microsoft.OpenApi.OData.Reader/Common/Constants.cs

```diff
@@ -110,19 +110,30 @@ internal static class Constants
         /// </summary>
         public static string ReferenceUpdateSchemaName = "ReferenceUpdateSchema";
 
+
+        /// <summary>
+        /// Name used for reference update.
+        /// </summary>
+        public static string ReferencePostSchemaName = "ReferencePostSchema";
```

Comment:
I'd use ReferenceUpdate and ReferenceCreate (or add) without the Schema suffix.
In Kiota this name is going to be used as the type name, and having schema in it makes for an ugly class name. Thoughts?

Reply:
We've had `ReferenceUpdateSchema` so I was following the same convention. So maybe `ReferenceCreateSchema`? I do not have a problem with appending the word schema. It provides clarity and prevents ambiguity.

Reply:
In the OpenApi Description yes, but in the code people will have to do something like

```CSharp

var myVal = new ReferenceCreateSchema();
```

Which feels odd, don't you think? Especially since it'd be so easy for us to make it.

```CSharp

var myVal = new ReferenceCreate();
```

Reply:
I see. Updating...